Notify/Notify/Notify/ViewModels/LoginPageViewModel.cs

```diff
@@ -94,5 +114,26 @@ private async void onSignUpClicked()
         {
             await Shell.Current.GoToAsync("///register");
         }
+
+        protected virtual void OnPropertyChanged(string propertyName)
+        {
+            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+        }
+        
+        private void storeUserCredentialsInPreferences()
+        {
+            if (!Preferences.ContainsKey("NotifyUserName"))
```

Comment:
Why not set the new value without checking it?
If the new credentials are different than the previous ones, it won't work.